develop.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [[ ! -d "docker" ]]; then
+    echo "This script must be run from the top level directory of the critiquebrainz-server source."
+    exit -1
+fi
+
+# invoke docker, but ungrep noisy influx log messages that I can't turn off
```

Comment:
this is a listenbrainz-specific comment (which is out of date anyway). It shouldn't be here.